### ISSUE TITLE
calcurse-caldav not downloading from Nextcloud

### ISSUE THREAD

**Issue by Htech155** [lfos/calcurse#189]
I am unable to get caldav to sync with my nextcloud server, I am uncertain where the problem stems from.
current config I have for caldav:
[General] 
Binary = calcurse 
Hostname = [hostname] 
Path = /remote.php/dav/calendars/[username]/ 
DryRun = No
InsecureSSL = No
SyncFilter = cal
Verbose = Yes
AuthMethod = basic
[Auth]
Username = [username]
Password = [password]

This current config when run with the --init=keep-remote does not give any errors, but does not download anything from the server. 

When I change the path to  /remote.php/dav/calendars/[username]/[calander] I get a http 400 error. 

On nextcloud I get an error as follows "Sabre\DAV\Exception\BadRequest: A calendar-query REPORT on a calendar with a Depth: 0 is undefined. Set Depth to 1"

**Comment by lfos** [comment]
Are you using a very old version of the calcurse-caldav script? This sounds like a bug that has been fixed back in late 2016; see issue #38 and commit e76b96c9 for details.

**Comment by Psycojoker** [comment]
FYI I've managed to make it work with nextcloud (and I have a "/" at the end of the PATH, not sure if that's important)

**Comment by matt-horwood-mayden** [comment]
you need to add a calendar name to the end of the path
`Path = /remote.php/dav/calendars/[username]/calendar/`

**Comment by GideonWolfe** [comment]
I have the following line 
`Path = /remote.php/dav/calendars/username/`

and I want to view all my calendars, not just one. I run 

`calcurse-caldav --init=keep-remote` 

because I don't want anything to corrupt my nextcloud calendar. I get the output 

```
Connecting to cloud.mydomain.com...
Removing all local calcurse objects...
Saving synchronization database to /home/me/.calcurse/caldav/sync.db...
0 items imported, 0 items removed locally.
0 items exported, 0 items removed from the server.
```
I'm not sure the correct way to do this

**Comment by lfos** [comment]
calcurse does not yet support multiple calendars. As of now, the only workaround is to sync each individual calendar to a different data directory and browse them individually with calcurse.

**Comment by GideonWolfe** [comment]
Fair enough. I'd love to see this feature, because nextcloud and other services sync multiple calendars. I'm currently using khal, which handles that fine but I prefer the UI of calcurse

**Comment by lfos** [comment]
Sure. The core problem is not the synchronization but moreso the fact that calcurse itself cannot handle multiple calendars yet.

**Comment by jforseth210** [comment]
This would be an awesome feature, I would love to be able to manage all of my calendars from calcurse. Thanks for all your hard work on this project!

**Comment by leana8959** [comment]
Hello,

I'm facing a similar problem: when using the endpoint `/remote.php/dav/principals/users/leana/`, `calcurse-caldav` succeeds but it imports 0 items from the remote.

This endpoint is indicated by the Nextcloud web interface. I do not mind using only one calendar. Is this a misconfiguration? How do I use only one calendar?

<details>
<summary>My configuration</summary>

```config
[General]
Binary = calcurse

Hostname = <redacted>
Path = /remote.php/dav/principals/users/leana/

AuthMethod = basic
InsecureSSL = No
HTTPS = Yes

SyncFilter = cal

DryRun = No
Verbose = Yes

[CustomHeaders]
Depth: 1

[Auth]
Username = leana
Password = <redacted>
```

</details>

Thank you for your time :)
